index.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
no need to put index.yaml in master branch, it will be generated in gh-pages branch

Reply:
done